collections/_sdk/data/note.md

```diff
@@ -119,6 +119,29 @@ commands_in_note = Note.commands.all()
 | is_patient_required                         | Boolean                                   |
 | allow_custom_title                          | Boolean                                   |
 
+### NoteStateChangeEvent
```

Comment:
How are these used? Could we add an example showing how to access them from a note? (Non-blocker as some info is better than none)

Reply:
Good call, added examples here: https://github.com/canvas-medical/documentation/pull/656/commits/a05d7e0bfd8fbe711704352a6728e5eb13facde5